experimental/torch_xla2/torch_xla2/functions.py

```diff
@@ -0,0 +1,29 @@
+"""Tensor constructor overrides"""
+import warnings
+
+import torch
+import jax.numpy as jnp
+from torch_xla2 import tensor
+
+fns = {
+  torch.tensor: jnp.array,
+  # torch.ones: jnp.ones,
+  # torch.zeros: jnp.zeros,
+  # torch.arange: jnp.arange,
+  # torch.linspace: jnp.linspace,
+  # torch.logspace: jnp.logspace,
+  # torch.empty: jnp.empty,
+  # torch.eye: jnp.eye,
+  # torch.full: jnp.full,
+}
+
+class XLAFunctionMode(torch.overrides.TorchFunctionMode):
+  def __torch_function__(self, func, types, args=(), kwargs=None):
+    jax_func = fns.get(func)
+    if not jax_func:
```

Comment:
if jax func dont exist, should just fall through with `return func(*args, **kwargs)`